### ISSUE TITLE
TMDB - Season poster doesn't download/show

### ISSUE THREAD

**Issue by nextlooper42** [jellyfin/jellyfin#4321]
**Describe the bug**
Scanner is not able to grab season posters from tmdb or even show them after scanning in manual poster search.

**System**
 - OS: Windows 10
 - Browser: Firefox 81
 - Jellyfin Version: 10.6.4

**To Reproduce**
1. Add show to library
2. Wait for scan
3. Season image that exists on TMDB doesn't load

**Expected behavior**
Every season will grab available image from TMDB according to language settings IF there is something to grab.

**Comment by cvium** [comment]
The season provider is disabled by default. Did you enable it for the library?

**Comment by nextlooper42** [comment]
Yes, i even tried reinstall. Doesn't work, curiously its working for TVDB.

**Comment by livingag** [comment]
Hi, I am getting this exact same issue (with TVDB working properly as well). This seems to be the corresponding error in the log:

```
[2021-01-26 11:03:28.375 +00:00] [ERR] [25] MediaBrowser.Providers.Manager.ProviderManager: "TmdbSeasonImageProvider" failed in GetImageInfos for type "Season"
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at MediaBrowser.Providers.Plugins.Tmdb.TV.TmdbSeasonImageProvider.GetImages(BaseItem item, CancellationToken cancellationToken)
   at MediaBrowser.Providers.Manager.ProviderManager.GetImages(BaseItem item, CancellationToken cancellationToken, IRemoteImageProvider provider, List`1 preferredLanguages, Nullable`1 type)
```

This seems to be only for Season images as far as I can tell, images for Movies and Series etc. loads fine from TMDB.

**Comment by livingag** [comment]
This is fixed in the latest stable release candidate (10.7.0-rc3).

Probably fixed by dc73d044de83110c28b33000cf0d5d8628f3d99e

**Comment by cvium** [comment]
Let us know if it's still a problem
